app/repository/nasa_api.js

```diff
@@ -0,0 +1,18 @@
+const axios = require('axios');
+
+async function fetchAsteroids(startDate, endDate) {
+    try {
+        const response = await axios.get(process.env.URL, {
+            params: {
+                start_date: startDate,
+                end_date: endDate,
+                api_key: process.env.API_KEY,
```

Comment:
Good way is to use som kind of config file to export env variables - you can map them in some structure, it will be easier to use it in case nested variables, e.g. `config.nasaClient.apiKey`

Reply:
Added! thank you for review❣